roles/server_create_delete/tasks/main.yml

```diff
@@ -43,17 +45,25 @@
   
   # Wait for the server to really start and become accessible
   - name: Wait for SSH port to become active
-    wait_for:
+    ansible.builtin.wait_for:
       port: 22
       host: "{{ server_ip }}"
-      timeout: 600
+      timeout: 300
     tags: ["az={{availability_zone}}", "service=compute", "metric=create_server"]
-  
+
+  - name: Wait another 5 sec for coreos
```

Comment:
I don't get this extra wait

Reply:
weird, but once CoreOS told me "system is not booted yet, only root can login", therefore add a tiny wait to give time to complete the boot